### DISCUSSION TITLE
New to Obtainium, having problems with adding Citra-Nightly

### DISCUSSION THREAD

**Discussion by soulcollectivex** [ImranR98/Obtainium#1145] [General]
Hey everyone, I'm new to Obtainium and I find it very useful for updating the apps I have on my phone, so I just wanted to see if anyone could help me with adding the latest updates of Citra-Nightly to Obtainium, I am able to add the GitHub page but the last known update it shows is Nightly-2006 and as of now the current one is actually Nightly-2051 as far as I have seen...

I have a suspicion that the reason Obtainium isn't able to get the latest updates for this version of the app is because Nightly-2006 is the last build where they had actual APK files in their releases, and every other update since then is still an APK, only it's just in a folder inside a ZIP file 😕

If anyone can help me figure out how to add the latest updates of citra nightly to Obtainium I would greatly appreciate it.

It would be cool if Obtainium could just download the zip file and unzip it  to a temp directory and install it that way but idk if that's too much to add to the app.. I have added some screenshots to show what I mean. Thanks 👍🏻 

![screenshot 1.jpg](https://github.com/ImranR98/Obtainium/assets/126836951/58a1148e-6b03-4b01-a113-267db3dd5f78)

![screenshot 2.jpg](https://github.com/ImranR98/Obtainium/assets/126836951/3b5c9d2a-c542-4981-b577-925cd53c703b)

![screenshot 3.jpg](https://github.com/ImranR98/Obtainium/assets/126836951/ce47a71a-e72c-4569-b387-8759b8e45fb3)

😶‍🌫️

**Comment by soulcollectivex** [comment]
So nobody unfortunately was able to discuss with me about the issue I was having with adding Citra-Nightly on Obtainium but now This issue seems to have been solved by the citra development team, they are now implementing APK files directly into their nightly releases since version nightly-2057 which is very helpful for a Obtainium novice like me. Have a good day/evening to whomever reads this✌🏻🫠

**Comment by ImranR98** [comment]
Missed this, glad to see you got it resolved (there is already an open issue to add zipped APK support).
